app/controllers/backend/financial_years_controller.rb

```diff
@@ -193,6 +203,18 @@ def run_progress
                                             step_label: progress_status[:label] }
     end
 
+    def prepare_for_closure
```

Comment:
Could this be extracted to its own controller ? Something like `ClosurePreparation`with `create` and `destroy` ?